packages/cli/command_help.pony

```diff
@@ -0,0 +1,182 @@
+use "buffered"
+
+primitive Help
+
+  fun general(cs: CommandSpec box): CommandHelp
+  =>
```

Comment:
Style-wise, the preceding line seems short enough that this double-arrow could be at the end of that line instead of having its own line here, without breaking the 80 columns rule.